### PR TITLE
Send button functionality invoice

### PR DESCRIPTION
**Pull request by ajinkyaa**
Save and send the invoice on the generate-invoice page.

- Code review pending.
- Will have a call with Apoorv regarding the QA PR review before the merge.

### CODE REVIEW

**Comment by apoorv1316** [comment]
@ajinkyaa Please take the latest pull from develop and re-run ci jobs

File: app/javascript/src/components/Invoices/Generate/Header.tsx [hunk 1]
```diff
@@ -1,98 +1,55 @@
...
+          <span className="ml-2 inline-block">SEND TO</span>
+        </button>
       </div>
-    </React.Fragment>
-  );
-};
+    </div>
+  </React.Fragment>
+);
+
```
Comment (JijoBose) [inline app/javascript/src/components/Invoices/Generate/Header.tsx:54]: Do fix these linting errors

File: app/javascript/src/components/Invoices/Generate/index.tsx [hunk 2]
```diff
@@ -4,10 +4,13 @@ import { useNavigate } from "react-router-dom";
 import { setAuthHeaders, registerIntercepts } from "apis/axios";
 import generateInvoice from "apis/generateInvoice";
 
+import invoicesApi from "apis/invoices";
 import Container from "./Container";
 import Header from "./Header";
+
+import { mapGenerateInvoice, unmapGenerateInvoice } from "../../../mapper/generateInvoice.mapper";
+import SendInvoice from "../modals/SendInvoice";
 import Invoice_settings from "./Invoice_settings";
```
Comment (JijoBose) [inline app/javascript/src/components/Invoices/Generate/index.tsx:13]: `./Invoice_settings` import should occur before import of `../../../mapper/generateInvoice.mapper`

**Review by JijoBose — CHANGES_REQUESTED** [review]
Do fix the `warnings` and `errors` from es-lint

**Comment by ajinkyaa** [comment]
@JijoBose Yeah sorry came in after conflicts, resolved.

**Review by JijoBose — APPROVED** [review]

**Comment by apoorv1316** [comment]
@ajinkyaa As discussed please send another pr for sending the invoice after saving it as a draft.

**Pull request merged**